### ISSUE TITLE
Independent Minimap Scroll resets when Code Glance is used

### ISSUE THREAD

**Issue by mrpeu** [olmokramer/atom-minimap-codeglance#22]
The new feature _Independent Minimap Scroll_ from minimap conflicts with _code glance_.
After scrolling the minimap, moving the pointer resets the scroll to align with the editor view. This does not happen when _code glance_ is disabled.

**Comment by perphyyoung** [comment]
I find this problem as well today. Wish there could be a way to see the _code glance_  wherever scrolling the minimap.

**Comment by rugk** [comment]
:+1: 

Any update on this?

**Comment by olmokramer** [comment]
Hi everyone, I'm sorry for the extremely late reply. As you have probably noticed I've stopped maintaining my Atom plugins a few years ago. I got very frustrated with how Atom was maintained and started ignoring all of my plugins. I'm sorry for that. I won't install Atom to test and mergeer PRs, but I'd be happy to transfer my repositories to anyone that wants to maintain any of my plugins. Again, I'm sorry.
